doc.go

```diff
@@ -455,6 +455,41 @@ of the returned value:
 	    }
 	}
 
+## Arrow batches
+
+There is a possibility to retrieve data in a columnar format similar to native server response.
+You have to use a proper `WithArrowBatches` context, example:
+
+	    var rows driver.Rows
+		err = conn.Raw(func(x interface{}) error {
+			rows, err = x.(driver.QueryerContext).QueryContext(ctx, query, nil)
+			return err
+		})
+
+		...
+
+		batches, err := rows.(sf.SnowflakeRows).GetArrowBatches()
+
+		... // use arrow records
+
+Limitations:
+
+1. When a query regards something that is not real data, it is returned usually in JSON format. It can't be used with arrow batches context. Examples: `SHOW PARAMETERS`.
+2. Snowflake handles timestamps in a range which is higher than available space in Arrow timestamp type. Because of that special treatment should be used (see below).
```

Comment:
Why do you use different capitalization for "arrow" (lowercase above, uppercase here)?

Consequently, you should handle timestamps as described in the following section.

Reply:
Good catch, changed to capital everywhere.